hyundai_kia_connect_api/KiaUvoApiEU.py

```diff
@@ -122,21 +122,24 @@ def __init__(self, region: int, brand: int, language: str) -> None:
             _LOGGER.warning(f"Unsupported language: {language}, fallback to en")
             language = "en"  # fallback to English
         self.LANGUAGE: str = language
+        self.brand: int = brand
```

Comment:
do we need to store this here?

Reply:
I need it in `_get_device_id`. Since that method is part of the class, I find it more natural to use class attributes.